fairseq/models/wav2vec/wav2vec2_asr.py

```diff
@@ -328,7 +328,7 @@ def __init__(self, args, tgt_dict=None):
             state = checkpoint_utils.load_checkpoint_to_cpu(
                 args.w2v_path, arg_overrides
             )
-            w2v_args = state["args"]
+            w2v_args = state["cfg"]["model"]
```

Comment:
can you change this to 
``` w2v_args = state.get("args", None) or state["cfg"]["model"] ```

thanks for the fix!

Reply:
Yup I did the same ``` state["cfg"]["model"] ```.
Happy to help😁

Reply:
great. please push your changes and then ill merge them

Reply:
Done!

Reply:
ah sorry i already merged this as part of another fix i was doing earlier today

Reply:
No issues.